fe/fe-core/src/main/java/org/apache/doris/catalog/BuiltinScalarFunctions.java

```diff
@@ -574,6 +580,8 @@ public class BuiltinScalarFunctions implements FunctionHelper {
             scalar(Sm3sum.class, "sm3sum"),
             scalar(Sm4Decrypt.class, "sm4_decrypt"),
             scalar(Sm4Encrypt.class, "sm4_encrypt"),
+            scalar(Sm4DecryptV2.class, "sm4_decrypt_v2"),
+            scalar(Sm4EncryptV2.class, "sm4_encrypt_v2"),
```

Comment:
please add function in the order of alphabet

Reply:
done